src/components/AudioTrack/layouts/Row.vue

```diff
@@ -77,8 +79,8 @@ export default {
     const isSmall = computed(() => props.size === 's')
 
     return {
+      seperator: this.$t('interpunct'),
```

Comment:
Also using `$t` outside computed will not be reactive.

Reply:
ah, yep. will fix

Reply:
Actually, I can't figure out how to use i18n with the composition api 😢

Reply:
We can put the $t inside the template.

Reply:
ooooh, thanks. super easy 😄